openedx/openedx-platform
# refactor: Update get block OLX view to support versions [FC-0062]

## Description

- Deprecate `get_block_draft_olx`
- Deprecate get olx view in content libraries
- Create `get_block_olx` in xblock API with support of versions
- Create get olx view in xblock 


## Supporting information

- Github issue: https://github.com/openedx/frontend-app-authoring/issues/1354#issuecomment-2491748690
- Used on: https://github.com/openedx/frontend-app-authoring/pull/1534
- Internal ticket: [FAL-3976](https://tasks.opencraft.com/browse/FAL-3976)

## Testing instructions

- Verify that the tests covers the new code
- Follow the testing instructions on https://github.com/openedx/frontend-app-authoring/pull/1534

## Deadline

ASAP
